scripts/run_integration_tests.sh

```diff
@@ -69,7 +69,7 @@ source $(dirname $0)/setup.sh || exit 1
 source $(dirname $0)/setup_gae.sh || exit 1
 
 # Install third party dependencies
-bash scripts/install_third_party.sh
+bash -x scripts/install_third_party.sh
```

Comment:
Thanks for the explanation! In that case I suggest adding a comment to explain this (at the places it occurs) -- I don't think people would find it out on their own.